ibis/backends/oracle/__init__.py

```diff
@@ -211,6 +387,15 @@ def _metadata(self, query: str) -> Iterable[tuple[str, dt.DataType]]:
         if isinstance(sg_expr, sg.exp.Table):
             sg_expr = sg.select(STAR).from_(sg_expr)
 
+        def transformer(node):
+            if isinstance(node, sg.exp.Table):
+                return sg.table(node.name, quoted=True)
+            elif isinstance(node, sg.exp.Column):
+                return sg.column(col=node.name, quoted=True)
```

Comment:
Oooh, I hadn't really thought about this but I suspect we should probably apply this transform in general to backends whose `quoted` value is `True`.

Reply:
Anyway, follow-up fodder.